charts/posthog/tests/ingress.yaml

```diff
@@ -32,3 +32,115 @@ tests:
           # Unfortunately we can't match the whole manifest as there are few
           # metadata values linked to the chart version
           path: spec
+
```

Comment:
This is not strictly part of this PR but it's just some additional unit tests for the ingress definition